test/compatibility_lifecycle/reference/versioned_feature_list.yaml

```diff
@@ -461,6 +461,10 @@
     lockToDefault: false
     preRelease: Alpha
     version: "1.32"
+  - default: false
```

Comment:
Should this be enabled by default?

Reply:
@pohly thoughts?

Reply:
Yes. ResourceClaim status was also enabled by default at beta. Such beta's then don't do much unless the DynamicResourceAllocation feature gate gets enabled, but that's fine.